vpr/src/util/vpr_utils.h

```diff
@@ -180,6 +180,9 @@ int get_physical_pin(t_physical_tile_type_ptr physical_tile,
                      t_logical_block_type_ptr logical_block,
                      int pin);
 
+int net_pin_tile_index(const ClusterNetId net_id, int net_pin_index);
+int pin_tile_index(const ClusterPinId pin);
```

Comment:
Since these are commonly used, should include give some comments describing what they do.

Reply:
Done